packages/solana/src/connectors/universalProvider.ts

```diff
@@ -1,11 +1,10 @@
-import UniversalProvider from '@walletconnect/universal-provider'
+import { UniversalProvider as Provider } from '@walletconnect/universal-provider'
 
 import { SolStoreUtil } from '../utils/scaffold/SolanaStoreUtil.js'
 import type { WalletConnectAppMetadata } from './walletConnectConnector.js'
-
+import type UniversalProvider from '@walletconnect/universal-provider'
```

Comment:
I guess this is the main fix? Was it because it's conflicting between the type definitions and the actual class while compiling?

Reply:
Main fix is ```import { UniversalProvider as Provider } from '@walletconnect/universal-provider'``` also split type import. [In the example ](https://www.npmjs.com/package/@walletconnect/universal-provider) you can see default import, but default import doesnot contain init function, i guess webpack issue. The problem was UniversalProvider imported from default did not contains init function.